File: 6_python/sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    number = get_int("Height: ")
    if number > 0 and number < 9:
        break

# start from 0 to number - 1
index = 1
while index <= number:
    print(("#" * index).rjust(number, " "), end="")
    print("  ", end="")
    print("#" * index)
    index += 1

File: 1_c/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height > 8 || height < 1);

    for (int i = 0; i < height; i++)
    {
        for (int j = height - 1; j >= 0; j--)
        {
            if (j <= i)
            {
                printf("#");
            }
            else
            {
                printf(" ");
            }
        }
        printf("  ");
        for (int j = 0; j < height; j++)
        {
            if (j <= i)
            {
                printf("#");
            }
        }
        printf("\n");
    }
}

